peter-piper.js

```diff
@@ -19,7 +19,14 @@ peterpiper += "Where's the peck of pickled peppers Peter Piper picked?"
 
 var counter = 0;
 
-// TODO: Count the number of times an uppercase P or lowercase p character
-//       occurs in the `peterpiper` string.
-
-console.log("There are " + counter + " P's in Peter Piper.");
\ No newline at end of file
+/*
+var p = peterpiper.match(/p/i) //is this cheating?
+counter = p.length-1 //Caause I feel like this is cheating.
```

Comment:
Definitely NOT cheating! It's also faster than looping through the string characters (you could use something like https://www.npmjs.com/package/spectrophotometer to verify).